src/test/scala/org/broadinstitute/dsde/workbench/sam/service/StatusServiceSpec.scala

```diff
@@ -73,16 +56,15 @@ class StatusServiceSpec extends AnyFreeSpec with Matchers with BeforeAndAfterAll
   private def failingDatabase = {
     // background database configured to connect to non existent database
     val dbReferenceOverride = DbReference(DatabaseNames.Background, TestSupport.blockingEc)
-    val service = new StatusService(directoryDAOWithAllUsersGroup(false), createLdapDaoWithConnectionPool(), NoExtensions, dbReferenceOverride, pollInterval = 10 milliseconds)
+    val service = new StatusService(directoryDAOWithAllUsersGroup(false), NoExtensions, dbReferenceOverride, pollInterval = 10 milliseconds)
     service
   }
 
   val cases = {
     Seq(
-      ("ok", ok, StatusCheckResponse(true, Map(OpenDJ -> SubsystemStatus(true, None), GoogleGroups -> SubsystemStatus(true, None), Database -> SubsystemStatus(true, None)))),
```

Comment:
TODO: might need to revert this if devops wants us to maintain opendj status endpoint api